MAVProxy/modules/mavproxy_arm.py

```diff
@@ -120,6 +120,25 @@ def cmd_disarm(self, args):
             0, # param6
             0) # param7
 
+    def all_checks_enabled(self):
+        ''' returns true if the UAV is skipping any arming checks'''
+        arming_mask = int(self.get_mav_param("ARMING_CHECK",0))
+        if arming_mask == 1:
+            return True
+        for bit in arming_masks.values():
+            if not arming_mask & bit and bit != 1:
+                return False
+        return True
+
+    def mavlink_packet(self, m):
+        mtype = m.get_type()
+        if mtype == 'HEARTBEAT' and m.type != mavutil.mavlink.MAV_TYPE_GCS:
```

Comment:
Yeah, I guess there won't be a problem. It would be good to have this corrected though, I'll look how to make days have 48 hours instead of 24, I need it too :smile: